packages/shared/profiles/sagas.ts

```diff
@@ -98,7 +103,7 @@ function* initialRemoteProfileLoad() {
   let profile: Avatar
 
   try {
-    profile = yield call(ProfileAsPromise, userId, undefined, isGuest ? ProfileType.LOCAL : ProfileType.DEPLOYED)
+    profile = yield call(ensureFriendProfile, userId, isGuest ? ProfileType.LOCAL : ProfileType.DEPLOYED)
```

Comment:
This one now breaks the logic now, rendering all the fallbacks below unreachable because the function generates a random profile if it can't be loaded from the server